### ISSUE TITLE
TV UX/UI

### ISSUE THREAD

**Issue by venkat2305** [recloudstream/cloudstream#286]
### Describe your suggested feature

1. While the video is being played, clicking the down button on the remote should move the cursor/pointer to the buttons under the progress bar and select one of them preferably, the first one from the left side. Clicking up button opens the media control page but clicking the down button doesn't do anything. It would be great if you could add this functionality. 
2. skipop button under the progress bar should be at starting of left. As we use this button for every single episode, it would be great if this button could be reached with least number of clicks.
3. Subtitles should be above the progress bar. Subtitles are overlapping with the progress bar as seen in this picture 
  
![Screenshot_2022-12-28-11-11-19-845_com lagradost cloudstream3 prerelease](https://user-images.githubusercontent.com/55329452/209764063-ef13a911-ac79-4def-8728-016d9d9a6deb.jpg)

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I will fill out all of the requested information in this form.

**Comment by LagradOst** [comment]
All of these a known 🗿
I am currently working on tv shit

**Comment by rokejulianlockhart** [comment]
What does #2 mean? I don't understand the sentence at all.

**Comment by venkat2305** [comment]
It's on tv. Right now you can see that the skip op button is towards the right end. To reach it on tv , it needs more clicks. So I'm asking th Devs to place it towards the left so it can be reached with less clicks

**Comment by rokejulianlockhart** [comment]
Oh, “Skip OP”. Autocorrect is the culprit, I imagine.

**Comment by venkat2305** [comment]
Do you watch the intro for every episode? I don't. 

> Is their position currently customisable?
No

**Comment by rokejulianlockhart** [comment]
In which case, @venkat2305, I believe that the ideal solution is allowing customization of the placement of such.

**Comment by venkat2305** [comment]
It's extra work and the devs already have so much to do. So i don't think we will see customisation options.

**Comment by LagradOst** [comment]
> In which case, @venkat2305, I believe that the ideal solution is allowing customization of the placement of such.

a lot of work and bad for android tv as you would need to program nextleftid ect programmatically

**Comment by rokejulianlockhart** [comment]
@venkat2305 and @LagradOst, I merely refer to customization of the order of placement. Anything more complex isn't of much use, usually.

**Comment by LagradOst** [comment]
> @venkat2305 and @LagradOst, I merely refer to customization of the order of placement. Anything more complex isn't of much use, usually.

ye, and that is complex because UI in android is a lot easier to do declarative and not programmatically

**Comment by allenbaby** [comment]
Yes, this feature is very useful for a seamless TV UI experience.
